internal/core/config/config.go

```diff
@@ -93,8 +93,7 @@ type Config struct {
 	// host.  If you set this to blank/null, the internal status server will
 	// not be started.
 	InternalStatusHost string `yaml:"internalStatusHost" default:"localhost"`
-	// The port on which the internal status server will listen.  See
-	// `internalMetricsHost`.
```

Comment:
I think this was supposed to be a reference to `internalStatusHost`, but the reason for the reference was so to better explain what that port was for.